src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/UWP.cs

```diff
@@ -262,12 +262,25 @@ public class Application : IProgram
             public string LogoPath { get; set; }
             public UWP Package { get; set; }
 
-            private int Score(string query)
+            // Function to calculate the score of a result
+            protected virtual int Score(string query)
```

Comment:
Yes, for a function to be overloaded it must be abstract or virtual. To test the Result function, the score function must be overloaded as it calls various methods from Wox.Infrastructure and throws a null reference error if it is not overridden.

Reply:
`UWP.Application` constructor requires `UWP` object which currently couldn't be mocked as it calls `InitializeAppInfo` which results in file read. But since that has been removed in #4521, it would be easy to mock `UWP` class and create a `UWP.Application` instance directly.

Reply:
@ryanbodrug-microsoft, what do you suggest we do? Do I remove the mocked class from this PR and add this test in once PR #4521 is in? There is an issue capturing that tests must be added to the UWP class (#4262).